{
    "id": "CWE-584",
    "name": "Return Inside Finally Block",
    "href": "AIP/quality-standards/CWE/items/CWE-584",
    "url": "https://cwe.mitre.org/data/definitions/584.html",
    "description": "The code has a return statement inside a finally block, which will cause any thrown exception in the try block to be discarded.",
    "isoPatterns": null,
    "count": 5,
    "qualityRules": {
        "name": "Quality Rules in numeric ID order",
        "href": "AIP/quality-standards/CWE/items/CWE-584/quality-rules"
    },
    "qualityTemplates": {
        "name": "Quality Templates in numeric ID order",
        "href": "AIP/quality-standards/CWE/items/CWE-584/quality-templates"
    }
}
